### ISSUE TITLE
Support instant apps

### ISSUE THREAD

**Issue by carstenhag** [Triple-T/gradle-play-publisher#529]
**Is your feature request related to a problem? Please describe.**
Currently, I am using this plugin to automatically release beta/alpha versions of my app, and then I manually promote them to the production track.

I then have to create a new release at the instant apps tab and select the lates release.

**Describe the solution you'd like**
Would be pretty handy if the plugin could release both normal and instant app at the same time.

**Describe alternatives you've considered**
Keep doing it manually, likely forgetting to update the instant app release once in a while

**Additional context**
Not many people are using instant apps yet, and they have been pretty buggy (graphic glitches) for me. Seems like Google is pushing them though.

**Comment by SUPERCILEX** [comment]
FYI: I talked with Googlers who work on the API at I/O and they said they're not working on this. 😢 Ple

**Comment by SUPERCILEX** [comment]
If you're interested in this, please file a support request asking them to add support for instant app uploads with the play publisher API.

**Comment by kenyee** [comment]
Done :-)
https://issuetracker.google.com/issues/138364068

**Comment by SUPERCILEX** [comment]
I've also filed a bug internally (yours is unlikely to get any attention unfortunately :cry:).

**Comment by CristianGM** [comment]
Aren't Instant Apps deprecated?
( They have been replaced by dynamic features )

**Comment by SUPERCILEX** [comment]
@CristianGM Yeah, but I'm pretty sure you still have to register them separately in the `#ManageInstantReleasesPlace` section of the Play Console.

**Comment by SmasSive** [comment]
Yes, you have to manage the instant apps releases separately but you can upload directly the app bundle and play store will serve the right instant app (contained in that bundle)
The problem then is that there is no access to this let's name it... "instant app track"?

**Comment by SUPERCILEX** [comment]
@SmasSive correct: as far as I'm aware, there's no API we can use to publish the app bundle as an instant app.

**Comment by SmasSive** [comment]
Hi!
Still no API? I've been reviewing the official documentation but I didn't see anything related to AIA, maybe I missed something?

On the other hand, any other alternative? Like uploading the AAB and then manually publishing it in the AIA track? Do you think it's doable?

**Comment by andreimesina** [comment]
Hi @SUPERCILEX , we have also tried releasing instant apps using the plugin, but with no success. 

Apparently at some point it was possible to use the plugin to upload the instant app bundle, without making a release. 
But currently, it is not even possible to upload the bundle, as the API gives the following error:

```
> A failure occurred while executing com.github.triplet.gradle.play.tasks.CommitEdit$Committer
   > 403 Forbidden
     POST https://androidpublisher.googleapis.com/androidpublisher/v3/applications/...
     {
       "code": 403,
       "errors": [
         {
           "domain": "global",
           "message": "You cannot rollout this release because it does not allow any existing users to upgrade to the newly added APKs.",
           "reason": "forbidden"
         }
       ],
       "message": "You cannot rollout this release because it does not allow any existing users to upgrade to the newly added APKs.",
       "status": "PERMISSION_DENIED"
     }
```

Do we know if Google is looking to support this at some point?

Thank you
